tests/installation/install_and_reboot.pm

```diff
@@ -104,6 +104,7 @@ sub run() {
         if (match_has_tag("yast2_package_retry")) {
             record_soft_failure "boo#1018262 - retry failing packages";
             send_key 'alt-y';    # retry
+            die "boo~1018262 - seems to hung on retry" unless wait_screen_change { sleep 4 };
```

Comment:
should be "boo#1018262". And "seems to hung on retry" sounds weird to me. "Seems to *be* hung"? "stuck"? "stalled"?